platforms/xamarin.ios/System.Runtime.Serialization.csproj

```diff
@@ -2,6 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <AssemblyOriginatorKeyFile>../silverlight.pub</AssemblyOriginatorKeyFile>
```

Comment:
You could also just add a dir.targets file in the directory and put it there. However I'm not worried about it either way.